### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by Tumworobere**
**For this milestone of your portfolio website, I implemented the following:**

   - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
   - When the user clicks (or taps) the close (X) button, the popup disappears.
   -  Popups for both mobile and desktop screen sizes

### CODE REVIEW

File: js/modal.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  document.querySelector('*').style.overflow = 'auto';
+  document.querySelector('#portfolio').classList.remove('activeblur');
+  document.querySelector('.about').classList.remove('activeblur');
+}
+
+// function toggle() {
+// const closed = document.querySelector('#close-btn');
+// closed.classList.toggle('hidden');
+// }
+// toggle();
```
Comment (HillaryOkello) [inline js/modal.js:102]: - [ ] Please remove the commented code to make your code look clean. Commented code makes your project look messy. If you need to review a previous version of your code, you can always use git history.

**Review by HillaryOkello — CHANGES_REQUESTED** [review]
# Hello @Tumworobere,

## To highlight

- Your readme is very professional ✅
- Your pull request description is descriptive ✅ 

Good job so far! You've passed almost all the checklists for this project.
There are a few issues that you still need to work on to go to the next project but you are almost there! You can do it 💪🏻 


## Required Changes ♻️
- [ ] The popup isn't responsive it does not show in the mobile version of the project, please fix it and make it appear like the one in the Figma template.
- Your design
pop

- Figma design
pop2
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Tumworobere, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Tumworobere** [comment]
> # Status: Approved ✔️ ✔️ ✔️
> ### Hi Tumworobere,
> ## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
> ## Happy Coding 👏👏👏
> #### Please leave any questions or comments in the PR if you have any doubts or unclarity!
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Thank you @SajjadAhmad14

**Pull request merged**